code/modules/plumbing/ducts.dm

```diff
@@ -37,120 +38,113 @@ All the important duct code:
 	var/list/neighbours = list()
 	///wheter we just unanchored or drop whatever is in the variable. either is safe
 	var/drop_on_wrench = /obj/item/stack/ducts
+	///Copy color and layer to drop_on_wrench when created. Must be /obj/item/stack/ducts or a subtype if used.
+	var/customize_drop_on_wrench = TRUE
```

Comment:
Curious, is there a reason for this var? Do you foresee anything needing to make it false at some point?